src/java/org/apache/cassandra/service/accord/AccordJournal.java

```diff
@@ -236,7 +236,7 @@ public boolean awaitTermination(long timeout, TimeUnit units) throws Interrupted
     /**
      * Auxiliary records are journal entries that aren't Accord protocol requests - such as {@link FrameRecord}.
      */
-    void appendAuxiliaryRecord(AuxiliaryRecord record, Object context)
+    void appendAuxiliaryRecord(AuxiliaryRecord record, FrameContext context)
```

Comment:
This won't be always `FrameContext` going forward (at the very least we are likely to store results of execution at some point).